MPF.Frontend/ViewModels/MainViewModel.cs

```diff
@@ -2211,29 +2211,61 @@ private bool ValidateBeforeDumping()
             var outputDirectory = Path.GetDirectoryName(_environment!.OutputPath);
             string outputFilename = Path.GetFileName(_environment.OutputPath);
 
-            // If a complete dump already exists
-            bool foundFiles = _environment.FoundAllFiles(outputDirectory, outputFilename);
-            if (foundFiles && _displayUserMessage != null)
+            // If a complete or partial dump already exists
+            bool foundAllFiles = _environment.FoundAllFiles(outputDirectory, outputFilename);
+            if (foundAllFiles && _displayUserMessage != null)
             {
                 bool? mbresult = _displayUserMessage("Overwrite?", "A complete dump already exists! Are you sure you want to overwrite?", 2, true);
                 if (mbresult != true)
                 {
                     LogLn("Dumping aborted!");
                     return false;
                 }
+                // else continue with dump, don't check further
             }
             else
             {
-                // If a complete dump exists from a different program
-                InternalProgram? programFound = _environment.CheckForMatchingProgram(outputDirectory, outputFilename);
-                if (programFound != null && _displayUserMessage != null)
+                // If a partial dump exists
+                bool foundAnyFiles = _environment.FoundAnyFiles(outputDirectory, outputFilename);
+                if (foundAnyFiles && _displayUserMessage != null)
                 {
-                    bool? mbresult = _displayUserMessage("Overwrite?", $"A complete dump from {programFound} already exists! Dumping here may cause issues. Are you sure you want to overwrite?", 2, true);
+                    bool? mbresult = _displayUserMessage("Overwrite?", $"A partial dump already exists! Dumping here may cause issues. Are you sure you want to overwrite?", 2, true);
                     if (mbresult != true)
                     {
                         LogLn("Dumping aborted!");
                         return false;
                     }
+                    // else continue with dump, don't check further
```

Comment:
Either capitalize the first letter or remove the comments after the `!= true` checks